AttitudeManager/FlightModes/Src/fbwa.cpp

```diff
@@ -0,0 +1,82 @@
+#include "constrain.h"
+#include "fbwa.hpp"
+#include "CommonDataTypes.hpp"
+#include "config.hpp"
+#include "AM.hpp"
+
+namespace AM {
+
+FBWA::FBWA()
+{
+    //Get PID tuning data from config
+    config::ControlTuning_t tuningData = config::flightmodes[config::FM_FBWA_IDX].tuningData;
+    
+    PIDController::PID PitchPIDValues = {
+        .kp = tuningData.PIDValues.pitchPID.p,
+        .kd = tuningData.PIDValues.pitchPID.d,
+        .ki = tuningData.PIDValues.pitchPID.i,
+        .i_max = tuningData.PIDValues.pitchPID.i_max,
+        .min_output = tuningData.PIDValues.pitchPID.min_output,
+        .max_output = tuningData.PIDValues.pitchPID.max_output
+    };
+    
+    PIDController::PID RollPIDValues = {
+        .kp = tuningData.PIDValues.rollPID.p,
+        .kd = tuningData.PIDValues.rollPID.d,
+        .ki = tuningData.PIDValues.rollPID.i,
+        .i_max = tuningData.PIDValues.rollPID.i_max,
+        .min_output = tuningData.PIDValues.rollPID.min_output,
+        .max_output = tuningData.PIDValues.rollPID.max_output
+    };
+
+    pitchPID_ = new PIDController(PitchPIDValues);
+    rollPID_ = new PIDController(RollPIDValues);
+}
+
+AttitudeManagerInput FBWA::run(const AttitudeManagerInput& input) {
+    // Get latest attitude from sensor fusion
+    SensorFusionOutput currentState = AttitudeManager::getSensorFusionData();
```

Comment:
Calling this here instead of receiving an argument because some flightmodes don't need sensor fusion.